### PR TITLE
[15.0][MIG]account_banking_sepa_direct_debit

### PR DESCRIPTION
**Pull request by manuelregidor**
Supersedes: https://github.com/OCA/bank-payment/pull/887
Depends on: https://github.com/OCA/bank-payment/pull/914

### CODE REVIEW

**Review by HaraldPanten — APPROVED** [review]
Functional review in local environment using https://github.com/OCA/bank-payment/pull/914 for testing as well

**Review by ValentinVinagre — APPROVED** [review]
LGTM 👍🏻

File: account_banking_sepa_direct_debit/models/account_payment_method.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+            "pain.008.003.02",
+        ]:
+            path = "account_banking_sepa_direct_debit/data/%s.xsd" % self.pain_version
+            return path
+        return super().get_xsd_file_path()
+
+    @api.model
+    def _get_payment_method_information(self):
+        res = super()._get_payment_method_information()
+        res["sepa_direct_debit"] = {"mode": "unique", "domain": [("type", "=", "bank")]}
```
Comment (ValentinVinagre) [inline account_banking_sepa_direct_debit/models/account_payment_method.py]: change `"mode": "unique"` for "`mode": "multi"`

**Review by ValentinVinagre — CHANGES_REQUESTED** [review]

**Review by ValentinVinagre — APPROVED** [review]
LGTM 👍🏻

File: account_banking_sepa_direct_debit/models/account_payment_method.py (hunk 1 above)
Comment (ValentinVinagre) [reply account_banking_sepa_direct_debit/models/account_payment_method.py]: Perfect, thanks 😄

**Pull request merged**